install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -20,10 +20,9 @@ items:
       heritage: {{ .Release.Service }}
       release: {{ .Release.Name }}
   spec:
-    type: {{ .Values.service.type }}
     ports:
-      - port: {{ .Values.service.externalPort }}
```

Comment:
does this service.externalPort mean to export zipkin/jaeger web console?
I think maybe .Values.service.* are for configuring the zipkin service, and .Values.zipkin.* & .Values.jaeger.* are for tracing service.
Not sure about that.

Reply:
We'd better keep this service port as externalPort.

Reply:
We'd better keep this service port as externalPort.

Reply:
we're try to prevent values.yaml flooded with more parameters, let's reuse current parameter.

Reply:
I agree with it. I have done a rollback.